src/app/services/date/date.service.ts

```diff
@@ -83,7 +83,7 @@ export class DateService {
 	 * and the last value is the last value of `DateRange`
 	 */
 	getDates(dateRange: DateRange): Date[] {
-		const [start, end] = dateRange;
+		const [start, end] = dateRange.map((d) => startOfDay(d));
```

Comment:
This prevents the error that `end` used to be 1 second before midnight and we used to get the following error.
By this fix the error if no longer persisted

<img width="700" alt="Screenshot 2022-11-02 at 15 48 04" src="https://user-images.githubusercontent.com/47821225/199506777-c6012c25-d78a-48e6-8f06-e1176980e151.png">

Reply:
I am okej with both implementation, don't really have a preference, but I would like to see in the console why we thrown the error

Reply:
sounds good, let's keep the console error but remove the safeguard

Reply:
this is outdated, the implementation/date correction was removed